CONTRIBUTING.md

```diff
@@ -1,145 +1,125 @@
 
-```bash
-$ git commit -m "fix(nav): incorrect URL for Travel nav item :P"
-```
+A clear and concise description of the enhancement to be added include a step-by-step guide if applicable.
+Add any other context or screenshots or animated GIFs about the feature request
 
-> Note that the _Bad_ example results in a longer commit header. This is partly attributed to the scope not being more specific and personal expression tacked on the end.
+### Describe alternatives you've considered
 
-**Note regarding subjects for bug fixes:**
+A clear and concise description of any alternative solutions or features you've considered.
 
-Summarize _what is fixed_, rather than stating that it _is_ fixed. The _type_ ("fix") already specifies the state of the issue.
+## Reporting bugs
 
-For example, don't do:
+All issues are submitted within GitHub issues. Please check this before submitting a new issue.
 
-```bash
-$ git commit -m "fix(nav): corrected Travel URL"
-```
+### Describe the bug
 
-Instead, do:
+A clear and concise description of what the bug is.
 
-```bash
-$ git commit -m "fix(nav): broken URL for Travel"
-```
+### Provide step-by-step guide on how to reproduce the bug
 
+Steps to reproduce the behavior, please provide code snippets or a link to repository
 
-#### Body and Footer (optional)
+### Expected behavior
 
-The body and footer should wrap at 80 characters.
+Please provide a description of the expected behavior
 
-The **body** describes the commit in more detail and should not be more than 1 paragraph (3-5 sentences). Details are important, but too much verbosity can inhibit understanding and productivity -- keep it clear and concise.
+### Screenshots
 
-The **footer** should only reference Pull Requests or Issues associated with the commit.
+If applicable, add screenshots or animated GIFs to help explain your problem.
 
-For bug fixes that address open issues, the footer should be formatted like so:
+### System information
 
-```bash
-Closes #17, #26
-```
-and for Pull Requests, use the format:
+Provide the system information which is not limited to the below:
 
-```bash
-Related #37
-```
+- OS: [e.g. macOS, Windows]
+- Browser (if applies) [e.g. chrome, safari]
+- Version of one-app-router: [e.g. 5.0.0]
+- Node version:[e.g 10.15.1]
 
-If a commit is associated with issues and pull requests, use the following format:
+### Security Bugs
 
-```bash
-Closes #17, #26
-Related #37
-```
-> Issues should always be referenced before pull requests, as shown above.
+Please review our [Security Policy](./SECURITY.md). Please follow the instructions outlined in the policy.
 
-#### Piecing It All Together
+## Getting in contact
 
-Below is an example of a full commit message that includes a header, body and footer:
+- Join our [Slack channel](https://one-amex.slack.com) request an invite [here](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjY)
```

Comment:
should this be added ? i'm not sure anyone are on these channels yet

Reply:
Yes, we have a few people in the team in the channel, i think it would be a good idea to start using and to get any one with questions asking in the channel

Reply:
if we have people in the channel then keep this